src/modules/emotionHandler/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+################################################################################
+#                                                                              #
+# Copyright (C) 2023 Fondazione Istitito Italiano di Tecnologia (IIT)          #
+# All Rights Reserved.                                                         #
+#                                                                              #
+################################################################################
+
+project(emotionHandler)
+cmake_minimum_required(VERSION 3.22)
+
+find_package(YARP 3.7.2 REQUIRED)
+find_package(OpenCV 4.5.4 REQUIRED)
+add_executable(${PROJECT_NAME})
+target_sources(${PROJECT_NAME} PUBLIC emotionHandler.cpp emotionHandler.h main.cpp)
+
+file(COPY ${CMAKE_CURRENT_SOURCE_DIR}/images DESTINATION ${CMAKE_BINARY_DIR})
+
+# Link the application with YARP and OpenCV libraries
+target_link_libraries(${PROJECT_NAME} ${YARP_LIBRARIES} ${OpenCV_LIBRARIES})
```

Comment:
I would prefer using CMake targets instead of vars, in this way you link only what you need instead linking **all** opencv and YARP libraries

It would become something like:
```
target_link_libraries(${PROJECT_NAME} YARP::YARP_os
                                                            YARP::YARP_dev
							    opencv_core 
                                                            opencv_highgui)
```

I am not sure if those are sufficient, in fact this approach is an exercise to be aware of what you are using